edb/server/server.py

```diff
@@ -1262,6 +1305,18 @@ async def _signal_sysevent(self, event, **kwargs):
             metrics.background_errors.inc(1.0, 'signal_sysevent')
             raise
 
+    def _on_remote_database_quarantine(self, dbname):
+        if not self._accept_new_tasks:
+            return
+
+        # Block new connections to the database.
+        self._block_new_connections.add(dbname)
+
+        async def task():
+            await self._pg_pool.prune_inactive_connections(dbname)
```

Comment:
Can you just pass self._pg_pool.prune_inactive_connections(dbname) to `create_task`?